src/Npgsql/NpgsqlSchema.cs

```diff
@@ -15,7 +15,7 @@ namespace Npgsql
     /// </summary>
     static class NpgsqlSchema
     {
-        public static DataTable GetSchema(NpgsqlConnection conn, string? collectionName, string?[]? restrictions)
+        public static Task<DataTable> GetSchema(NpgsqlConnection conn, string? collectionName, string?[]? restrictions, bool async = true, CancellationToken cancellationToken = default)
```

Comment:
Please remove the default value for async (here and below), we don't do that anywhere else